intg/src/main/java/org/apache/atlas/model/typedef/AtlasStructDef.java

```diff
@@ -109,6 +111,12 @@ public List<AtlasAttributeDef> getAttributeDefs() {
         return attributeDefs;
     }
 
+    public static void validateTypeName(AtlasStructDef type) throws AtlasBaseException {
```

Comment:
Can we keep this in a common and standard place to avoid writing same function again and again